src/client/components/OrganizationPanel.js

```diff
@@ -1,24 +1,64 @@
-import React from 'react'
-import styled from 'styled-components'
-import { Text } from '@aragon/ui'
+import React, { useContext } from 'react'
 import CardWrapper from '../wrappers/styleWrappers/CardWrapper'
-
-const addMore = () => console.log('add more orgs')
+import { useProfile } from '../hooks'
+import { ModalContext } from '../wrappers/modal'
+import EducationHistoryTile from './EducationHistoryTile'
+import { open, removeItem } from '../stateManagers/modal'
+import { Text, theme } from '@aragon/ui'
+import styled from 'styled-components'
 
 const OrganizationPanel = () => {
+  const { organizations, viewMode } = useProfile()
+  const { dispatchModal } = useContext(ModalContext)
+
+  const organizationsNotEmpty = Object.keys(organizations).length > 0
+
+  const cardProps = {
+    title: 'Organizations',
+    addMore: organizationsNotEmpty
+      ? () => dispatchModal(open('organizations'))
+      : null,
+    addSeparators: true,
+    viewMode,
+  }
+
   return (
-    <CardWrapper title="Organisations" addMore={addMore}>
-      <Center>
-        <Text color="grey">Coming Soon</Text>
-      </Center>
+    <CardWrapper {...cardProps}>
+      {organizationsNotEmpty ? (
+        Object.keys(organizations).map(id => (
+          <EducationHistoryTile
```

Comment:
If you're going to reuse this component, please change the name of the component, so it makes more sense.

Reply:
I wasn't going to leave it, it was to be until `Render single organization info (logic) #8` is completed. However, it makes sense: I created a basic component on which #8 will be based.